test/closet/routes/multi-method-route.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+    method: ['get', 'post'],
```

Comment:
```suggestion
    // This case is significant because we should not auto-assign an
    // id to routes with multiple methods, as it is not allowed by hapi.
    method: ['get', 'post'],
```